packages/clerk-js/src/utils/captcha/CaptchaChallenge.ts

```diff
@@ -39,7 +40,7 @@ export class CaptchaChallenge {
    * Important: For this to work at the moment, the instance needs to be using SMART protection
    * as we need both keys (visible and invisible) to be present.
    */
```

Comment:
Is this comment still relevant?

Reply:
But this is now used for everyone, both for instances that have invisible and smart selected. This comment indicates that this function will fail if SMART is not selected on the Clerk Dashboard.